### PR TITLE
added Google Keep Clone in Vuejs

### PR DESCRIPTION
**Pull request by Maheshwari-Love**
Closes:  #762

- **Title** : Google Keep clone
- **Name:** :  Maheshwari Love 
- **Idenitfy yourself:** : GSSOC

### How Has This Been Tested? ⚙️

Locally

### Screenshots 📷

image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [ ] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Note to reviewers 📄

Added without before get assigned as today is last day of contribution please merge and assign me

### CODE REVIEW

**Comment by Maheshwari-Love** [comment]
@Avdhesh-Varshney can you please merge this

File: Vue-JS-Projects/Intermediate/Google-Keep/image.png
Comment (Avdhesh-Varshney) [inline Vue-JS-Projects/Intermediate/Google-Keep/image.png:1]: convert this into `webp` and named it to `screenshot`

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Vue-JS-Projects/Intermediate/Google-Keep/image.png
Comment (Maheshwari-Love) [reply Vue-JS-Projects/Intermediate/Google-Keep/image.png:1]: @Avdhesh-Varshney  added the changes

**Review by Avdhesh-Varshney — APPROVED** [review]

**Comment by Avdhesh-Varshney** [comment]
Please fill this survey form: https://forms.gle/Wvsp9N8TDF1WPStq9

**Pull request merged**